StripeApplePay/StripeApplePay/Source/Extensions/PKPayment+Stripe.swift

```diff
@@ -10,12 +10,12 @@ import PassKit
 
 extension PKPayment {
     /// Returns true if the instance is a payment from the simulator.
-    @_spi(STP) public func stp_isSimulated() -> Bool {
+    @_spi(STP) public func stp_ap_isSimulated() -> Bool {
```

Comment:
This works around a conflict with another stp_isSimulated() function in Xcode 16.

Reply:
supernit: Does ap stand for Apple Pay? can we expand to apple_pay for clarity